app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -26,6 +32,19 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     val error: LiveData<String> = mutableError
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
+    val eventSpeakers: MutableLiveData<List<Speaker>> = MutableLiveData()
```

Comment:
Remove, use the LiveData from DAO. What's the use of saving in database if just fetching from network

Reply:
Still not removed. You don't need to create a mutable livedata, you already have a livedata from dao